os/options.go

```diff
@@ -68,6 +70,10 @@ func (parser *EnvOptionParser) Options() (*agent.Options, error) {
 		options.EdgeInsecurePoll = true
 	}
 
+	if os.Getenv(EnvKeyEdgeTunnel) == "0" {
```

Comment:
its worth remembering that this code also becomes easier with #228 (not that this specific example is complex, it just adds to the cognitive load.